concepts/booleans/about.md

```diff
@@ -0,0 +1,15 @@
+# About
+
+The boolean type `Bool` is an enumeration of `True` and `False`. The basic boolean operators are `&&` (and), `||` (or), and `not`.
+
+```haskell
+True || False           -- True
+True && False           -- False
+```
+
+The three boolean operators each have a different operator precedence. They are evaluated in this order: `not` first, `&&` second, and finally `||`. If you want to bypass these rules, you can enclose a boolean expression in parentheses (`()`), as the parentheses have an even higher operator precedence.
```

Comment:
We probably want this section in the `introduction.md` too, as student will have to know about it in order to successfully finish the corresponding exercise.

Reply:
Done.